calblueprint/ks-web
# myles/farm profile graphs

[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?
- Added graphs to FarmProfile
- Used Moment.js to generate date labels.
- Setup for data fetching pipeline

### How to Review
- Update Airtable to newest schema
- See if graphs are loaded correctly
- Create Recent Harvest Logs.

### Related PRs
#61 

### Next steps
- Add airtable data, render filters
- Fix responsive issues (gap certification bar)

### Screenshots
https://user-images.githubusercontent.com/35586281/114339347-d1b14a00-9b09-11eb-91c7-24b40a9a8710.mp4

![image](https://user-images.githubusercontent.com/35586281/114339714-b561dd00-9b0a-11eb-80d9-490fff1cb106.png)
(vertical monitor bby)

CC: @phoebeli23
